Source/WebCore/rendering/TextBoxPainter.cpp

```diff
@@ -329,12 +329,15 @@ void TextBoxPainter<TextBoxPath>::paintForegroundAndDecorations()
 
             bool shouldPaintDraggedContent = !(m_paintInfo.paintBehavior.contains(PaintBehavior::ExcludeSelection));
             if (shouldPaintDraggedContent) {
-                auto markedTextsForDraggedContent = MarkedText::collectForDraggedContent(m_renderer, m_selectableRange);
+                auto markedTextsForDraggedContent = MarkedText::collectForDraggedAndTransparentType(DocumentMarker::Type::DraggedContent, m_renderer, m_selectableRange);
                 if (!markedTextsForDraggedContent.isEmpty()) {
                     shouldPaintSelectionForeground = false;
                     markedTexts.appendVector(markedTextsForDraggedContent);
                 }
             }
+            auto markedTextsForTransparentContent = MarkedText::collectForDraggedAndTransparentType(DocumentMarker::Type::TransparentContent, m_renderer, m_selectableRange);
+            if (!markedTextsForTransparentContent.isEmpty())
+                markedTexts.appendVector(markedTextsForTransparentContent);
```

Comment:
Nit -

```suggestion
                markedTexts.appendVector(WTFMove(markedTextsForTransparentContent));
```

(We could also fix the existing code above)